modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -225,14 +288,52 @@ function testSurveyAccountsAddSurvey()
         $this->safeFindElement(
             WebDriverBy::Name("PSCID")
         )->sendKeys("8889");
+        $this->safeFindElement(
+            WebDriverBy::Name("VL")
+        )->sendKeys($invalidVisitLabel);
+        $this->safeFindElement(
+            WebDriverBy::Name("Test_name")
+        )->sendKeys($instrument);
+        $this->safeFindElement(
+            WebDriverBy::Name("fire_away")
+        )->click();
+        $bodyText =  $this->safeFindElement(
+            WebDriverBy::cssSelector(".error")
+        )->getText();
+        $this->assertContains(
+            "You must specify a valid Visit Label.",
+            $bodyText
+        );
+
+        // Ensure that an error is thrown a visit label is supplied for which
+        // the candidate does not have a session.
+        // NOTE This test will work only so long as the Candidate defined in
+        // this file does not have a visit $visitLabel.
+        $this->safeGet($this->url . "/survey_accounts/");
+        $btn = self::$add;
+        $this->webDriver->executescript(
+            "document.querySelector('$btn').click()"
+        );
+        $this->safeFindElement(
+            WebDriverBy::Name("CandID")
+        )->sendKeys("999999");
+        $this->safeFindElement(
+            WebDriverBy::Name("PSCID")
+        )->sendKeys("8889");
+        $this->safeFindElement(
+            WebDriverBy::Name("VL")
+        )->sendKeys($visitLabel);
+        $this->safeFindElement(
+            WebDriverBy::Name("Test_name")
+        )->sendKeys($instrument);
         $this->safeFindElement(
             WebDriverBy::Name("fire_away")
         )->click();
         $bodyText =  $this->safeFindElement(
             WebDriverBy::cssSelector(".error")
         )->getText();
         $this->assertContains(
-            "Visit V1 does not exist for given candidate",
+            "You must specify a valid Visit Label.",
             $bodyText
         );
```

Comment:
This is the exact same code as 255-277, so this is also causing an integration test failure. I suggest either removing this section or changing the error message back to `"Visit V1 does not exist for given candidate"`